src/database/config/database.js

```diff
@@ -2,8 +2,18 @@ import Sequelize from 'sequelize';
 import { config } from 'dotenv';
 
 config();
+let db_url;
 
-const dbase = new Sequelize(process.env.DB_CONNECT, {
+if (process.env.NODE_ENV == 'development') {
+  db_url = process.env.DEV_DB;
+}
+if (process.env.NODE_ENV == 'test') {
+  db_url = process.env.TEST_DB;
+} else {
+  db_url = process.env.DB_CONNECT;
+}
```

Comment:
Find a better way to do this without hardcoding the environments as you already have them in NODE_ENV ("development","test","production")